library/ovirt_disk.py

```diff
@@ -0,0 +1,768 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2016 Red Hat, Inc.
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
```

Comment:
Can you please add a comment here asking to remove it once we will require ansible 2.8?